documentation/book/appendix_metrics.adoc

```diff
@@ -79,14 +80,52 @@ When an alert condition is evaluated as true on the Prometheus server, it sends
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-service-account.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-service-account.yaml
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml > prometheus-operator-cluster-role.yaml
+
+Next, apply all the files using the following commands:
+[source,shell,subs="+quotes"]
+kubectl apply -f prometheus-operator-service-account.yaml
+kubectl apply -f prometheus-operator-cluster-role.yaml
+kubectl apply -f prometheus-operator-cluster-role-binding.yaml
+kubectl apply -f prometheus-operator-deployment.yaml
+
+
+The {ProductName} repository contains configuration files for the Prometheus server. When you apply these configuration files, the following resources are created in your Kafka cluster and managed by the Prometheus Operator.
+
+* A `ClusterRole` that grants permissions to read the Prometheus health endpoints of the Kubernetes system, including cAdvisor and the kubelet for container metrics. The Prometheus server configuration uses the Kubernetes service discovery feature in order to discover the pods in the cluster from which it gets metrics.  For this feature to work correctly, the service account used for running the Prometheus service pod must have access to the API server so it can retrieve the pod list.
+* A `ServiceAccount` for the Prometheus pods to run under.
+* A `ClusterRoleBinding` which binds the aforementioned `ClusterRole` to the `ServiceAccount`.
+* A `Deployment` to manage the Prometheus Operator pod.
+* A `ServiceMonitor` to manage the configuration of the Prometheus pod.
+* A `Secret` to manage additional Prometheus settings.
+* A `PrometheusRule` to manage alert rules for the Prometheus pod.
+* A `Prometheus` to manage the configuration of the Prometheus pod.
+
+Prometheus also provides an alerting system through the link:https://prometheus.io/docs/alerting/alertmanager/[Alertmanager] component.
+To enable alerting, the provided link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml[`prometheus-rules.yaml`] file describes a `PrometheusRule` resource that defines sample alerting rules for Kafka and Zookeeper metrics.
+When an alert condition is evaluated as true on the Prometheus server, it sends the alert data to the Alertmanager which then uses the configured notification methods to notify the user.
+
+For more information about setting up alerting rules, see https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[Alerting Rules] in the Prometheus documentation.
+
+
+==== Deploying Prometheus on {OpenShiftName}
```

Comment:
It doesn't break anything, but you can remove the plus character from the heading.

Reply:
The `+` character is not in the pushed changes. That is patch syntax :)

Reply:
D'oh! OK, that's fine with me.